RecoHGCal/TICL/plugins/TrackstersMergeProducer.cc

```diff
@@ -531,22 +531,20 @@ void TrackstersMergeProducer::energyRegressionAndID(const std::vector<reco::Calo
 
 void TrackstersMergeProducer::assignTimeToCandidates(std::vector<TICLCandidate> &resultCandidates) const {
   for (auto &cand : resultCandidates) {
-    if (cand.tracksters().size() > 1) {  // For single-trackster candidates the timing is already set
-      float time = 0.f;
-      float timeErr = 0.f;
-      for (const auto &tr : cand.tracksters()) {
-        if (tr->timeError() > 0) {
-          auto invTimeESq = pow(tr->timeError(), -2);
-          time += tr->time() * invTimeESq;
-          timeErr += invTimeESq;
-        }
+    float time = 0.f;
+    float timeErr = 0.f;
+    for (const auto &tr : cand.tracksters()) {
+      if (tr->timeError() > 0) {
+        auto invTimeESq = pow(tr->timeError(), -2);
+        time += tr->time() * invTimeESq;
+        timeErr += invTimeESq;
       }
-      if (timeErr > 0) {
-        timeErr = 1. / timeErr;
+    }
+    if (timeErr > 0) {
+      timeErr = 1. / timeErr;
```

Comment:
Being a bit picky, but it's rather confusing to keep the same notation for a quantity and its inverse. 
I think it would be better to simply keep calling it something like`invTime` throughout, and then only reconvert at the last moment, i.e., `cand.setTimeError(sqrt(1/invTimeError))` or something along those lines.